### PR TITLE
PMM-7408 rules duplicate delete

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm-app/tests/ia/pages/alertRulesPage.js [hunk 1]
```diff
@@ -108,9 +120,19 @@ module.exports = {
     closeModal: '$modal-close-button',
     addRule: '$add-alert-rule-modal-add-button',
     cancelAdding: '$add-alert-rule-modal-cancel-button',
-    // editAlertRule returns Edit template button locators for a given source
+    cancelDelete: '$cancel-delete-modal-button',
+    delete: '$confirm-delete-modal-button',
+    // showDetails returns Show rule details button locator for a given rule name
+    showDetails: (ruleName) => `${module.exports.elements.rulesNameCell(ruleName)}//button[@data-qa="show-alert-rule-details"]`,
```
Comment (atymchuk) [inline pmm-app/tests/ia/pages/alertRulesPage.js:126]: Having to refer to `module.exports` seems a bit verbose, we could figure out a shorter syntax.

**Review by atymchuk — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]
Thank you :+1:

File: pmm-app/tests/ia/pages/alertRulesPage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/alertRulesPage.js:126]: I wasn't able to find a shorter syntax. Could you please point me?

Comment (atymchuk) [reply pmm-app/tests/ia/pages/alertRulesPage.js:126]: Sure, you need to write a helper function `rulesNameCell` and use it everywhere :)

Comment (yurkovychv) [reply pmm-app/tests/ia/pages/alertRulesPage.js]: done, thanks a lot!

**Pull request merged**